spec/tapioca/addon_spec.rb

```diff
@@ -0,0 +1,59 @@
+# typed: true
+# frozen_string_literal: true
+
+require "addon_spec_helper"
+require "ruby_lsp/ruby_lsp_rails/runner_client"
+require "minitest/hooks"
+
+module RubyLsp
+  module Tapioca
+    class AddonSpec < Minitest::HooksSpec
+      # The approach here is based on tests within the Ruby LSP Rails gem
+
+      # TODO: Replace by `before(:all)` once Sorbet understands it
+      def initialize(*args)
+        super(*T.unsafe(args))
+        FileUtils.cp("spec/dummy/bin/rails", "bin/rails")
+        @outgoing_queue = Thread::Queue.new
+        @client = T.let(nil, T.nilable(RubyLsp::Rails::RunnerClient))
+        FileUtils.chdir("spec/dummy") do
+          @client = RubyLsp::Rails::RunnerClient.new(@outgoing_queue)
+        end
+      end
+
+      after(:all) do
+        # TODO: Remove `bind` once Sorbet understands `after(:all)`
```

Comment:
Sorbet can already understand it. We just need a signature for it (which is a nice contribution to rbi-central).

```ruby
class Minitest::HooksSpec
  sig { params(arg: Symbol, block: T.proc.bind(T.attached_class).void).void }
  def self.after(arg, &block); end
end
```

Reply:
https://github.com/Shopify/rbi-central/pull/300